op-e2e/e2eutils/disputegame/helper.go

```diff
@@ -38,7 +38,7 @@ const alphabetGameDepth = 4
 var lastAlphabetTraceIndex = big.NewInt(1<<alphabetGameDepth - 1)
 
 // rootPosition is the position of the root claim.
-var rootPosition = faultTypes.NewPositionFromGIndex(1)
+var rootPosition = faultTypes.NewPositionFromGIndex(common.Big1)
```

Comment:
```suggestion
var rootPosition = faultTypes.NewPositionFromGIndex(big.NewInt(1))
```

Sorry about all the back and forward on this...